### ISSUE TITLE
"syndesis install" accidentally fails when Server URL contains "oc"

### ISSUE THREAD

**Issue by nobusue** [syndesisio/syndesis#4123]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
"syndesis install" accidentally fails when Server URL contains "oc".

```
tools/bin/commands/util/openshift_funcs: line 394: [: Sererhttps://master: integer expression expected
tools/bin/commands/util/openshift_funcs: line 396: [: Sererhttps://master: integer expression expected
```

This is caused by:
https://github.com/syndesisio/syndesis/blob/b2dc4f3e9866bd7e17210e54cb27efe49b2ee79c/tools/bin/commands/util/openshift_funcs#L9

It seems to be fixed to, for example

```
oc version | grep "^oc" | tr -d oc\ v | cut -f1 -d "+"
```

to avoid unintentional match.

Problematic case (similar to my environment):

```
$ oc version
oc v3.11.0+0cbc58b
kubernetes v1.11.0+d4cacc0
features: Basic-Auth

Server https://master.ocp311.example.com:8443
openshift v3.11.16
kubernetes v1.11.0+d4cacc0
```

Under this condition, version gather logic doesn't work correctly.

```
$ oc version | grep oc | tr -d oc\ v | cut -f1 -d "+"
3.11.0
Sererhttps://master.p311.nsue.mbi:8443
```

**Comment by zregvart** [comment]
@nobusue thanks for reporting the issue! I think your proposed solution would fix this. Are you working on a pull request also?

**Comment by zregvart** [comment]
Fixed via #4124
